### DISCUSSION TITLE
Token generation using refresh token [Jul 05]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2660] [Discord Threads]
> 🙋‍♂️ **Manh-Trung BUI**

**Comment by o1lab** [comment]
> 👨‍💻 **pranavxc**  
#### Currently it's not possible in this way  
#  
> 👨‍💻 **pranavxc**  
#### we are keeping `refresh_token` in httpOnly cookie  
#  
> 👨‍💻 **pranavxc**  
#### and the api expects `refresh_token` in the cookie  
#  
> 👨‍💻 **pranavxc**  
#### To pass cookie with axios you can set `withCredentials : true`    
####    
#### https://stackoverflow.com/a/43178070  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/993455836607295560)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
